sql/moz-fx-data-shared-prod/telemetry_derived/adm_engagements_daily_v1/metadata.yaml

```diff
@@ -0,0 +1,17 @@
+---
+friendly_name: adMarketplace Daily Engagements
+description: >
+  Click Volume, partitioned by day with the following facets:
+    - submission_date
+    - phase
+    - country
+    - placement
+    - partner
+    - version
+    - normalized_channel
+owners:
+  - tbrooks@mozilla.com
+labels:
+  application: firefox
+  schedule: daily
+  incremental: true
```

Comment:
AFAICT, this query is not incremental (though we should make it so).

Reply:
yep yep, totally agree, just didn't know how to make that happen :)